lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -97,7 +97,6 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
 
                 try:
                     crt = CopyResultTb(
-                        self.context.controller,
```

Comment:
Don't we also need to remove the `--controller` option at line 179 below?

Reply:
Making the result tarball still needs to know where the result directory is, so that switch still has value (it is used to locate non-standard directory trees).

Reply:
that is an optional option. That is basically used for storing the controller metadata and verifying that in metadata.log file.

Reply:
Why is this option only available on move and not push?

Reply:
Because `-move` creates the tarball while `-push` only uploads it?  My understanding is that we need to know the location (and/or be given the controller to add to the metadata) when we create the tarball, but, once the tarball exists, the controller is "baked in", so `-push` doesn't need (or want) to know about it.  (But...I could be confused....)